Distribution/Server/Users/Types.hs

```diff
@@ -67,8 +80,19 @@ instance Text UserName where
 isValidUserNameChar :: Char -> Bool
 isValidUserNameChar c = (c < '\127' && Char.isAlphaNum c) || (c == '_')
 
+instance Migrate UserInfo where
```

Comment:
Yeah, so stick the old data decl just above, down here at the bottom of the file, just above the migration.